compiler/noirc_frontend/src/hir_def/traits.rs

```diff
@@ -148,18 +149,6 @@ impl PartialEq for Trait {
 }
 
 impl Trait {
-    pub fn set_methods(&mut self, methods: Vec<TraitFunction>) {
-        self.methods = methods;
-    }
-
-    pub fn set_trait_bounds(&mut self, trait_bounds: Vec<ResolvedTraitBound>) {
-        self.trait_bounds = trait_bounds;
-    }
-
-    pub fn set_where_clause(&mut self, where_clause: Vec<TraitConstraint>) {
-        self.where_clause = where_clause;
-    }
```

Comment:
I don't really mind but why remove these methods?

Reply:
Oh, it's because now there's `visibility` so I initially created a mutable method for it but then I noticed it's `pub` (I made it `pub` like the others) and then noticed all of them are pub so I didn't understand why we needed mutable methods if the fields are already mutable. But maybe it's to make it clear they are mutable, not sure. I'll put them back.